src/NzbDrone.Core/Notifications/NotificationService.cs

```diff
@@ -172,5 +173,31 @@ public void Handle(HealthCheckFailedEvent message)
                 }
             }
         }
+
+        public void Handle(MovieFileDeletedEvent message)
+        {
+            if (message.Reason != MediaFiles.DeleteMediaFileReason.Upgrade)
```

Comment:
Maybe we should move the skip logic to trakt connection and pass this in the message. Users may want to see update deletes on other notifications?

Reply:
I'm almost thinking we should implement different items in NotificationEventItems.js above

For example have: name="onDelete"
              helpText="On Movie File Deletion for any reason"

and name="onDeleteExceptUpgrade"
              helpText="On Movie File Deletion except On Upgrade"

Then make trakt Support OnDeleteExceptUpgrade, other connection types can only Support the ones they need

Reply:
I'm not really sure what the best way to do it is. I just don't like making it special for trakt.. as the helpText should be clear and the connections should really behave the same across connection types.. i.e. OnDelete should be the same whether it is for Trakt Connection or Plex Connection (if it was implemented there). if we have OnDeleteExceptOnUpgrade, it would be consistent.